src/main/java/core/basesyntax/operation/ReturnOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {

```

Comment:
There should be no empty line at the beginning of the class implementation. Please remove the empty line after the class declaration to follow the common practices.